doc/release/0.16.0-notes.rst

```diff
@@ -26,6 +26,10 @@ New features
 
 The function `scipy.linalg.invpascal` computes the inverse of a Pascal matrix.
 
+Added two extra wrappers for LAPACK least-square solvers. Namely, they are 
+*gelsd and *gelsy. Added new parameter to linalg.lstsq which sets the
```

Comment:
ReST formatting: code should get double backticks around it. Especially when it uses ``*``, because otherwise that'll be interpreted as special formatting for italic or bold text.